components/orderForm/OrderDetails.js

```diff
@@ -109,7 +118,9 @@ export default function OrderDetails({children}) {
   return (
     <>
       <h2 className="h-10 text-lg mb-2 block tracking-wide text-gray-600 font-bold">Order Details</h2>
-
+      <div className="flex flex-row mb-3 space-x-5"> 
+        {inventoryInfo.map(infoStr => <ReactMarkdown className="mb-4 text-zinc-900 flex-grow border p-2 border-black rounded" components={smallMarkdownStyle} children={infoStr}></ReactMarkdown>)}
```

Comment:
```suggestion
        {inventoryInfo?.map(infoStr => <ReactMarkdown className="mb-4 text-zinc-900 flex-grow border p-2 border-black rounded" components={smallMarkdownStyle} children={infoStr}></ReactMarkdown>)}
```

we're getting an error because of the comment above - that should be fixed, but we can also do this to prevent the whole code from breaking

the `inventoryInfo?.` basically tells the code to only do the map function if inventoryInfo is defined, so we don't break trying to do `undefined.map()`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining